### ISSUE TITLE
HTML CSS

### ISSUE THREAD

**Issue by nathalie1226** [DavNej/Assignment_Buzzquiz#6]
Your CSS is nice but some the text of the question does not look as nice as the rest

**Comment by DavNej** [comment]
Yeah my cousin took the quizz and told me the same thing... but on my computer it is perferctly aligned... dont understand... is there a way to make sur it looks nice anywhere?
